packages/jellyfish-wallet-mnemonic/src/hd_node.ts

```diff
@@ -53,6 +53,16 @@ export class MnemonicHdNode implements WalletHdNode {
     return node.publicKey
   }
 
+  /**
+   * @return Promise<Buffer> uncompressed public key
+   */
+  async publicKeyUncompressed (): Promise<Buffer> {
```

Comment:
@thedoublejay if this wallet type isn't supported, we can't support Light Wallet.

Reply:
We have to find ways to convert compressed to uncompressed or expose the internals to support that,

Reply:
yea lemme think about that 🤔🤔🤔🤔

Reply:
resolved at https://github.com/BirthdayResearch/jellyfishsdk/pull/2105/commits/b72830cda4c6350514e48160d9f708b68042d86e